src/components/Forward/Forward.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import classNames from 'classnames'
+
+const propTypes = {
+  className: PropTypes.string,
+  scrollToRef: PropTypes.shape({ current: PropTypes.instanceOf(Element) })
+}
+
+const Forward = ({ className, scrollToRef, ...attributes }) => {
+  const classes = classNames(className, 'forward')
+  return (
+    <a
+      className={classes}
+      onClick={() =>
+        scrollToRef.current.scrollIntoView({
+          behavior: 'smooth',
```

Comment:
Mind that this is not supported by IE (ok ,fair) and `Edge` pre-chromium version: https://caniuse.com/#search=scrollIntoView .
Is it ok @francescozaia ?

Reply:
Actually, we're still supporting IE11 and early versions of Edge, as [they still sum up to a ~ 6% - 7% of usage in Italy](https://gs.statcounter.com/browser-version-market-share/all/italy), especially among public administrations.
So, I 'm sorry to be a pain, but I would use something else (or just some graceful degradation positioning for those browsers).

Reply:
From what I can see here, scrollIntoView is partially supported by IE11 and Edge pre-chromium, the part that is not supported is the smooth behaviour.

Reply:
If the animation type is not a concern then it's ok.

Reply:
I checked this on virtualbox, for both Internet explorer 11 and pre-chromium edge. Scrolling works fine, but happens instantly, i.e. without any animation with the existing `scrollIntoView`

Reply:
Great then! Good for me, thanks @Shreyans for checking! 👍